docs/develop/build-a-dApp/build-smart-contracts/EVM-extensions-precompiles.md

```diff
@@ -2,5 +2,39 @@
 sidebar_position: 3
 ---
 
-# EVM Extension (Precompiles)
+# EVM Extension (Precompiled Smart Contracts)
 
+## Overview
+
+Precompiled smart contracts are smart contracts
+that are built into the Ethereum Virtual Machine (EVM).
+Each offers specific functionality, that can be used by other smart contracts.
+Generally, they are used to perform operations that are either not possible
+or would be too expensive to perform
+with a regular smart contract implementation,
+such as hashing, elliptic curve cryptography, and modular exponentiation.
+
+By adding custom precompiled smart contracts to Ethereum's basic feature set,
+Evmos allows developers to use previously unavailable functionality in smart contracts,
+like staking and governance operations.
+This will allow more complex smart contracts to be built on Evmos
+and further improves the interoperability between Cosmos and Ethereum.
+It also is a key feature to achieve Evmos' vision
+of being the definitive dApp chain, where any dApp can be deployed once
+and users are able to interact with a wide range of different blockchains natively.
+
+## Available Precompiled Contracts
+
+The following precompiled contracts are available in Evmos' implementation of the EVM:
+
+| Address | Name            | Type | Stateful | EIP                                               |
+|---------|-----------------|------|----------|---------------------------------------------------|
+| 0x01    | ECRecover       |      | No       |                                                   |
```

Comment:
Let's update the address to be the full address (all zeroes)

Reply:
will do! what did you mean by "Type" btw? It was mentioned to be added in the Linear ticket but I wasn't sure what that was referring to

Reply:
wether it's a Ethereum precompile (the defaults that we're using now) or an Evmos precompile (developed by us)

Reply:
but we can remove the column for now I guess